cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -220,31 +233,46 @@ impl RawManifest {
 
         options_table.decor_mut().set_prefix(indoc! {r#"
 
-            # Additional options can be set in the `[options]` section. Refer to
-            # manifest.toml(5) for a list of available options.
+
+            ## Other Environment Options -----------------------------------------
         "#});
 
         // `systems` array
-        let mut systems_key = Key::new(MANIFEST_SYSTEMS_KEY);
-        systems_key.leaf_decor_mut().set_prefix(indoc! {r#"
-
-                # An environment that works on one system is guaranteed to work on the same type
-                # of system, but other systems may not have the same packages available, etc.
-                # In order to use the environment on a system you must explicitly add it to the
-                # `options.systems` list.
-            "#});
+        let default_systems_vec = [
+            "aarch64-darwin".to_string(),
+            "aarch64-linux".to_string(),
+            "x86_64-darwin".to_string(),
+            "x86_64-linux".to_string(),
+        ];
+        let systems_vec: &[&String] = if systems.is_empty() {
+            &default_systems_vec.iter().collect::<Vec<_>>()[..]
+        } else {
+            systems
+        };
+        let mut systems_array = Array::new();
+        for system in systems_vec {
+            let mut item = Value::from(system.to_string());
+            item.decor_mut().set_prefix("\n  "); // Indent each item with two spaces
+            if Some(system) == systems_vec.last() {
+                item.decor_mut().set_suffix(",\n"); // Add a newline before the first item
+            }
+            systems_array.push_formatted(item);
+        }
```

Comment:
We were aware we are defaulting to all systems, but the purpose of this initial manifest is to show what all is possible. Especially now when a lot of users are being onboarded. This will probably change in the future, but for now we only have few opportunities to teach users. Generally, there is always a conflict between advance and new users, those who use flox longer will find this manifest pointless, those who are just starting will (hopefully) find it more useful. For now, we decided to cater to new users, but I could easily see this change in the future.

@ysndr Is there a constant that provides available systems that I should use instead?

Reply:
there is `DEFAULT_SYSTEMS_STR` but i'd say here its a bit late to apply defaults, in fact we already _do_ provide defaults in the one single caller of this method in `PathEnvironment::init`.

Reply:
Also do we need to insist on this custom formatting? considering that a toml formatter with default settings would promptly reformat the systems line?

Reply:
Yes, vertical way of listing array is requested preferred few times. Also, this allows you to easily comment out the system that is not supported. Most importantly, it is much easier to eye-parse vertically than horizontally. 

If formatters break this, then those formatters are broken, they should respect new lines from users.